### DISCUSSION TITLE
Pulling prompt/completions via API

### DISCUSSION THREAD

**Discussion by mukundt** [langfuse/langfuse#1114] [Ideas]
### Describe the feature or potential improvement

Right now it's impossible to export a large (>60k rows) dataset of prompt/completions for finetuning via UI; would love to be able to do this programatically.

### Contribute

- [ ] Yes, I can implement this and raise a PR

**Comment by marcklingen** [comment]
Hi @mukundt, this is a known issue and will be fixed soon (streaming of larger exports to S3).

The API does offer exports of all generations via [`GET /api/public/observations`](https://api.reference.langfuse.com/#tag/Observations/operation/observations_getMany)
